### ISSUE TITLE
Elgonated graphics in Firefox browsers only

### ISSUE THREAD

**Issue by snowbound** [dimdenGD/OldTweetDeck#147]
Graphics started showing up elongated with the paperclip also being large. This is in Firefox Developer. Seems to be fine in Chromium.

![image](https://github.com/dimdenGD/OldTweetDeck/assets/596147/17b01a9c-ab93-4c16-ba2b-b8241abdeecc)

**Comment by snowbound** [comment]
Yup same here regarding refreshing the browser tab the graphics display properly.  That no doubt pushes me closer to being throttled, which happens every day now

**Comment by fREAST** [comment]
Got the same issues here but only now when they swapped to the x.com domain.
https://twitter.com/i/tweetdeck still works fine, and link previews are the correct size, but on https://x.com/i/tweetdeck they are broken.
Firefox with OldTweetDeck 3.6.2 and BetterTweetDeck 4.11.3.
Left is x.com, right is twitter.com.
![image](https://github.com/dimdenGD/OldTweetDeck/assets/8655676/6f0df22b-35d0-4a1f-a45a-e90bba924ca3)
